src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java

```diff
@@ -459,7 +459,7 @@ private void logAndUpdateDriveSubsystemStates() {
                 states.velocityYMPS = getRobotRelativeSpeeds().vyMetersPerSecond;
                 states.totalVelocity = Math.hypot(states.velocityXMPS, states.velocityYMPS);
                 states.angularVelocity = Units.radiansToDegrees(relativeRobotSpeeds.omegaRadiansPerSecond);
-                states.gyroAngleDegrees = gyro.getYaw().in(Degrees);
+                states.gyroAngleDegrees = gyro.getYaw(true).in(Degrees);
```

Comment:
Thinking about this again, I think it would be better to do true 

poseEstimator.updateWithTime(Timer.getFPGATimestamp(),
                                Rotation2d.fromRadians(gyro.getYaw(false).in(Radians)),
                                new SwerveModulePosition[] {
                                                frontLeft.getPosition(),
                                                frontRight.getPosition(),
                                                rearLeft.getPosition(),
                                                rearRight.getPosition()
                                });